tests/utils.go

```diff
@@ -500,7 +500,7 @@ func CreateBackupForResources(ctx context.Context, backupName, resources, snapsh
 
 func DeleteBackup(ctx context.Context, backupName string) error {
 	args := []string{
-		"delete", "backup", backupName,
+		"delete", "backup", "backupName",
```

Comment:
don't think we need " around the backupName otherwise it will try to delete backup named "backupName"

Reply:
done